services/src/datasets/external/nature40.rs

```diff
@@ -254,6 +276,31 @@ impl Nature40DataProvider {
         }
     }
 
+    async fn get_time_positions(
```

Comment:
- don't read the wcs description file twice but parse labels and positions simulatenously